app/src/componentLibrary/fieldTextFlex/fieldTextFlex.scss

```diff
@@ -0,0 +1,79 @@
+/*!
+ * Copyright 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@mixin additional-text($color) {
+  display: block;
+  font-family: $FONT-ROBOTO-REGULAR;
+  font-size: 11px;
+  line-height: 16px;
+  color: $color;
+}
+
+.text-area {
+  width: 100%;
+  min-height: 72px;
+  padding: 10px 12px;
+  border: none;
+  outline: 1px solid $COLOR--e-200;
```

Comment:
safari has issue with outline property with border-radius it shows with right angle, please use border property to cover all devices
<img width="759" alt="image" src="https://user-images.githubusercontent.com/42434528/170307936-8b7b1fc2-e70f-428a-85b1-ba22293157ec.png">